contents/articles/2016-11_pupil-lsl-plugin/index.md

```diff
@@ -0,0 +1,31 @@
+--- 
+title: Pupil plugin for Lab Streaming Layer
+date: Fri Nov 18 2016 13:37:50 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Introducing Pupil LSL relay plugin that relays pupil data, gaze data and notifications to the lab streaming layer..."
+featured_img: ""
+featured_img_thumb: "" 
+---
+
+The [Lab Streaming Layer](https://github.com/sccn/labstreaminglayer) (LSL) is a system that provides unified collection of measurement time series between programs, computers and devices over a network for distributed signal transport, time synchronization and data collection. LSL has an extensive range of supported measurement modalities including eye tracking. 
+
+Introducing [Pupil LSL relay plugin](https://github.com/sccn/labstreaminglayer/tree/master/Apps/PupilLabs). A [Pupil Capture](https://github.com/pupil-labs/pupil/wiki/Pupil-Capture) plugin that relays pupil data, gaze data and notifications to the Lab Streaming Layer which users can link the data to other inlets in the network.
```

Comment:
Move this to first paragraph.

Change first sentence to:
"We are excited to introduce the [Pupil + Lab Streaming Layer relay plugin]...